docs/changelog/undistributed/changelog_nd_20210805110350.rst

```diff
@@ -0,0 +1,3 @@
+
+* ND
```

Comment:
```suggestion
* ND (Nexus Dashboard)
```

Reply:
Changed, but with a twist:
It made more sense to me to have <long name> (<short name>) eg. Nexus Dashboard (ND)